src/serverrpc.cpp

```diff
@@ -56,20 +56,31 @@ CServerRpc::CServerRpc ( CServer* pServer, CRpcServer* pRpcServer, QObject* pare
     /// @rpc_method jamulusserver/getClients
     /// @brief Returns the list of connected clients along with details about them.
     /// @param {object} params - No parameters (empty object).
-    /// @result {array} result.clients - The list of connected clients.
+    /// @result {number} result.connections - The number of active connections.
+    /// @result {array}  result.clients - The list of connected clients.
     /// @result {number} result.clients[*].id - The client’s channel id.
     /// @result {string} result.clients[*].address - The client’s address (ip:port).
     /// @result {string} result.clients[*].name - The client’s name.
     /// @result {number} result.clients[*].jitterBufferSize - The client’s jitter buffer size.
     /// @result {number} result.clients[*].channels - The number of audio channels of the client.
+    /// @result {number} result.clients[*].instrumentCode - The id of the instrument for this channel.
+    /// @result {number} result.clients[*].instrumentName - The text name of the instrument for this channel.
```

Comment:
I think we could argue to also remove this, as it's a Jamulus protocol ID. But to get this PR going, I'd accept it for now.